modules/basegl/src/processors/volumeprocessing/volumeglprocessor.cpp

```diff
@@ -40,8 +40,8 @@ namespace inviwo {
 VolumeGLProcessor::VolumeGLProcessor(std::shared_ptr<const ShaderResource> fragmentShader,
                                      bool buildShader)
     : Processor()
-    , inport_("inputVolume")
-    , outport_("outputVolume")
+    , inport_("inputVolume", Document{"Input volume"})
```

Comment:
any reason for using `Document{}` instead of `_help`?

Reply:
This is probably from before I added _help